src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestReader.cs

```diff
@@ -317,7 +317,7 @@ private static List<ManifestFile> ReadFilesList(XElement xElement)
                 files.AddRange(srcElement.Value.Trim(';').Split(';').Select(s => 
                     new ManifestFile
                     {
-                        Source = s.SafeTrim().TrimStart(slashes),
```

Comment:
This was presumably intended for Windows, correct? What is the effect to Windows when it is removed?